### ISSUE TITLE
Prometheus integration

### ISSUE THREAD

**Issue by GopherJ** [casbin/casbin#540]

**Comment by hsluoyz** [comment]
@GopherJ what's Prometheus? How to integrate? adapter, middleware, or ? Please provide some details.

**Comment by hsluoyz** [comment]
If we didn't find someone to work on it, we can add "help wanted" label to attract contributors.

**Comment by nodece** [comment]
Add Prometheus can collect the number of function calls, response number of function. 

It is important for monitoring Casbin.

**Comment by GopherJ** [comment]
@hsluoyz Yes we can mark it as `help wanted` it's low priority but I think many people/adopters will probably need this feature to be able to understand better what's happening.

**Comment by GopherJ** [comment]
I may move this to `casbin-dashboard` issue later because it's better to be alive there
